### PR TITLE
ncm-resolver: merge config-{rpm,common} into config.pan

### PR DESCRIPTION
**Pull request by ajf8**

### CODE REVIEW

File: ncm-resolver/src/main/pan/components/resolver/config.pan [hunk 1]
```diff
@@ -5,5 +5,12 @@
...
-include { 'components/${project.artifactId}/config-rpm' };
+include 'components/${project.artifactId}/schema';
+include 'pan/functions';
+
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+prefix '/software/components/${project.artifactId}';
+
+'active' ?= true;
+'dispatch' ?= true;
```
Comment (stdweird) [inline ncm-resolver/src/main/pan/components/resolver/config.pan]: pre dependency on spma?

Comment (ajf8) [inline ncm-resolver/src/main/pan/components/resolver/config.pan]: There wasn't one originally, I'll add it in.

Comment (stdweird) [inline ncm-resolver/src/main/pan/components/resolver/config.pan]: @ajf8 i was not suggesting you forgot it, but while you are doing some (much appreciated) cleanup, why not fix it.

**Comment by stdweird** [comment]
LGTM, awaiting a rebase

**Pull request merged**